src/main/java/no/fint/betaling/util/CustomerFileGroup.java

```diff
@@ -0,0 +1,20 @@
+package no.fint.betaling.util;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import no.fint.betaling.model.CustomerGroup;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class CustomerFileGroup {
+    CustomerGroup foundCustomers;
+    List<String> notFoundCustomers;
+
+    public boolean hasCustomers(){
+        return foundCustomers.getCustomers().size() != 0 || notFoundCustomers.size() != 0;
```

Comment:
Use `.isEmpty()` instead, for some implementation of `List`, the `.size()` operation actually traverses the data to count it ...

Reply:
Done!